server-kotlin/src/main/resources/application.properties

```diff
@@ -17,8 +17,15 @@ city-request.appName=CityVizor
 city-sync.instanceUrls.prague=https://cityvizor.praha.eu:8080
 # Fulltext search
 fulltextSearch.indexRefreshEnabled=true
-# How often service which downloads budgets runs
-internet.stream.service.configuration.frequency=PT60M
+# How often service which downloads budgets runs, once per day
+internet.stream.service.configuration.frequency=PT1440M
 # Initial delay before service starts, this ensures that service is configured
 # and connected to db
 internet.stream.service.configuration.initDelay=PT5M
+# map of configuration, key is url of city as defined in app.profiles, column url
```

Comment:
I would prefer to not start it by default. We can move it to a separate properties file (application-citysync.properties) and include it using Spring profiles (https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-profiles)

Reply:
Mimochodem, proč to chceš zapínat všude?